### PR TITLE
Home gallery

### PR DESCRIPTION
**Pull request by dstephens3**

### CODE REVIEW

**Review by dantoth24 — CHANGES_REQUESTED** [review]
image
Something is creating a scroll bar along the bottom of the screen. I'm not sure how it appears on your computer, but this should not be able to happen. Something is larger than it should be, check your css.

**Comment by dstephens3** [comment]
I reduced the size of the photos and improved the previous and next buttons and added a link to their facebook page

File: app/views/home/home.html.erb [hunk 1]
```diff
@@ -1,4 +1,208 @@
 <div class="container-fluid">
```
Comment (dantoth24) [inline app/views/home/home.html.erb]: <div class="container-fluid" style="padding-top: 20px; padding-left: 30px">

Comment (dantoth24) [inline app/views/home/home.html.erb]: `<div class="container-fluid" style="padding-top: 20px; padding-left: 30px">`
Replace with this

File: app/views/home/home.html.erb [hunk 2]
```diff
@@ -1,4 +1,208 @@
...
 	<br>
-	<b>Welcome to the Women in Chemistry Initiative Website!</b><br>
+	<h2>Welcome to the Women in Chemistry Initiative Website!</h2><br><br>
 </div>
```
Comment (dantoth24) [inline app/views/home/home.html.erb]: Remove this `</div>`

File: app/views/home/home.html.erb [hunk 3]
```diff
@@ -1,4 +1,208 @@
...
+
+
+
+  this.next("expandedImg");
+}
+</script>
+
+
+
+</body>
```
Comment (dantoth24) [inline app/views/home/home.html.erb:67]: After `</body>` add `</div>`

**Review by dantoth24 — CHANGES_REQUESTED** [review]
Make these changes and confirm it looks correct on your computer. This fixes the horizontal scaling issues. We should also make sure to discuss in the retro how to organize css and javascript, I think generally it is not a good thing to keep those in the view, although my experience with this is limited.

**Review by dantoth24 — APPROVED** [review]
That fixed the issue, you're good to merge this.

File: app/views/home/home.html.erb [hunk 4]
```diff
@@ -1,4 +1,210 @@
-<div class="container-fluid">
+<div class="container-fluid" style="padding-top: 20px; padding-left: 30px">
 	<br>
-	<b>Welcome to the Women in Chemistry Initiative Website!</b><br>
+	<h2>Welcome to the Women in Chemistry Initiative Website!</h2><br><br>
+
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
```
Comment (harperiscoding) [inline app/views/home/home.html.erb]: styling/head stuff should not be in html.erb files

File: app/views/home/home.html.erb [hunk 5]
```diff
@@ -1,4 +1,210 @@
...
+'https://scontent-sjc3-1.xx.fbcdn.net/v/t31.0-8/16113169_1670482029911457_5672124951285453401_o.jpg?_nc_cat=103&_nc_ht=scontent-sjc3-1.xx&oh=3db7135f5c73eb990a8998380efb3584&oe=5D7779BC',
+'https://scontent-sjc3-1.xx.fbcdn.net/v/t1.0-9/45006054_1978141279145529_6321221626599833600_o.jpg?_nc_cat=111&_nc_ht=scontent-sjc3-1.xx&oh=c2d5b3fe73353b56e7fb0951315cfa02&oe=5D650991',
+'https://scontent-sjc3-1.xx.fbcdn.net/v/t31.0-8/14612434_1635012100125117_2082643130819550668_o.jpg?_nc_cat=100&_nc_ht=scontent-sjc3-1.xx&oh=7a39da2b628f9fd505874a3f280b9ee8&oe=5D2A3F9A',
+'https://scontent-sjc3-1.xx.fbcdn.net/v/t31.0-8/30945287_1859210474371944_5647595661945196635_o.jpg?_nc_cat=104&_nc_ht=scontent-sjc3-1.xx&oh=d32715a378b1b084c9899323c27a271c&oe=5D38D1EF',
+'https://scontent-sjc3-1.xx.fbcdn.net/v/t1.0-9/55532077_2058335077792815_6997238389384675328_o.jpg?_nc_cat=109&_nc_ht=scontent-sjc3-1.xx&oh=5c8cb5b826183d4ab484e21fc01a4b5e&oe=5D346A4A',
+'https://scontent-sjc3-1.xx.fbcdn.net/v/t31.0-8/22791912_1781779578781701_2304727614290918746_o.jpg?_nc_cat=110&_nc_ht=scontent-sjc3-1.xx&oh=6a229eb4c1a61a4bc0ef5de48a7e3d00&oe=5D696319']);
```
Comment (harperiscoding) [inline app/views/home/home.html.erb]: neither should long scripts

**Review by harperiscoding — CHANGES_REQUESTED** [review]

File: app/views/home/home.html.erb (hunk 5 above)
Comment (dstephens3) [reply app/views/home/home.html.erb]: I have removed everything but the photo urls. If these are in the javascript file then they do not get recognized for some reason

File: app/views/home/home.html.erb [hunk 6]
```diff
@@ -1,4 +1,85 @@
...
+	<input type='button' class="button" value='WICI Facebook Page' name='link' onclick="location.href='https://www.facebook.com/BerkeleyWICI/';"/>
+	</div>
+</div>
+<br>
+<br>
+<br>
+<br>
+
+<script>
+var arr = new collection(arr);
```
Comment (townsend3721) [inline app/views/home/home.html.erb:64]: I see you put this in the js.file, good job

File: app/views/home/home.html.erb [hunk 7]
```diff
@@ -1,4 +1,85 @@
...
+<div class="column">
+	<img id=7 src="https://scontent-sjc3-1.xx.fbcdn.net/v/t31.0-8/22791912_1781779578781701_2304727614290918746_o.jpg?_nc_cat=110&_nc_ht=scontent-sjc3-1.xx&oh=6a229eb4c1a61a4bc0ef5de48a7e3d00&oe=5D696319" alt="" style="width:95%" onclick="myFunction(this);">
+</div>
+</div>
+
+<div class="row">
+	<div class="column">
+	</div>
+	<div class="column">
+	<input type='button' class="button" value='WICI Facebook Page' name='link' onclick="location.href='https://www.facebook.com/BerkeleyWICI/';"/>
```
Comment (townsend3721) [inline app/views/home/home.html.erb]: There is already a link in the footer, so you can delete this

**Review by townsend3721 — APPROVED** [review]

**Pull request merged**